src/components/Common/GeneralDashboardSidebar/GeneralDashboardSidebar.jsx

```diff
@@ -20,17 +21,19 @@ const GeneralDashboardSidebar = ({ userDetail, sidebarItems }) => {
     return (
         <DashboardSidebarContainer isOpen={isOpen}>
             <UserProfile isOpen={isOpen}>
-                <div className="user-profile-image">
-                    <img
-                        style={{
-                            width: "40px",
-                            height: "40px",
-                        }}
-                        src={avatar}
-                        alt={userDetail?.username + " Profile Picture"}
-                    />
-                    <BiSolidCircleIcon />
-                </div>
+                <Link to={`/user/${userDetail?.username}`}>
+                    <div className="user-profile-image">
+                        <img
+                            style={{
+                                width: "40px",
+                                height: "40px",
+                            }}
+                            src={avatar}
+                            alt={userDetail?.username + " Profile Picture"}
+                        />
```

Comment:
I would also use Template literals here for consistency : 

`${userDetail?.username} Profile Picture}`

Reply:
good idea